timezone_converter/search_view.py

```diff
@@ -0,0 +1,34 @@
+from difflib import get_close_matches
+from typing import List
+
+from timezone_converter.helper import Helper
+
+
+class SearchView(Helper):
+    def __init__(
+        self,
+        search: str,
+    ) -> None:
```

Comment:
This should fit in a single line, and it is more readable that way (Black should be happy too...)